test/widgets/board_editor_test.dart

```diff
@@ -0,0 +1,200 @@
+import 'package:flutter/material.dart';
+import 'package:dartchess/dartchess.dart' as dc;
+import 'package:flutter_test/flutter_test.dart';
+import 'package:chessground/chessground.dart';
+
+const boardSize = 200.0;
+const squareSize = boardSize / 8;
+
+void main() {
```

Comment:
You can wrap all the tests in a `group('BoardEditor', () { ... })`.

So it will prefix all test names with `BoardEditor`.